streampipes-extensions/streampipes-connect-adapters/src/main/java/org/apache/streampipes/connect/GeneralAdaptersExtensionModuleExport.java

```diff
@@ -28,35 +28,42 @@
 import org.apache.streampipes.connect.adapters.ti.TISensorTag;
 import org.apache.streampipes.connect.adapters.wikipedia.WikipediaEditedArticlesAdapter;
 import org.apache.streampipes.connect.adapters.wikipedia.WikipediaNewArticlesAdapter;
-import org.apache.streampipes.extensions.management.model.SpServiceDefinition;
-import org.apache.streampipes.extensions.management.model.SpServiceDefinitionBuilder;
-import org.apache.streampipes.service.extensions.ExtensionsModelSubmitter;
+import org.apache.streampipes.extensions.api.connect.StreamPipesAdapter;
+import org.apache.streampipes.extensions.api.declarer.IExtensionModuleExport;
+import org.apache.streampipes.extensions.api.migration.IModelMigrator;
+import org.apache.streampipes.extensions.api.pe.IStreamPipesPipelineElement;
 
-public class ConnectAdapterInit extends ExtensionsModelSubmitter {
+import java.util.Collections;
+import java.util.List;
 
-  public static void main(String[] args) {
-    new ConnectAdapterInit().init();
+public class GeneralAdaptersExtensionModuleExport implements IExtensionModuleExport {
+
+  @Override
+  public List<StreamPipesAdapter> adapters() {
+    return List.of(
+        //new GdeltAdapter(),
```

Comment:
I guess we can remove the comments here, right? I would also suggest removing the adapters.

Reply:
I would suggest that we open an issue to deprecate some of the adapters in the `streampipes-connect-adapters` module for the upcoming release, so that we can remove them in 0.94.0. 
The module has several adapters that are out of date.

Reply:
Yes, we can do that in a separate PR